Strassens_Algorithm/Strassens_Algorithm.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Linq;
+
+
+public class Test
+{
+	public static void Main()
+	{
+		// your code goes here
+		int[,] firstMatrix = new int[2, 2];
+		int[,] secondMatrix = new int[2, 2];
+		int[,] productMatrix= new int[2, 2];
+		
+		Console.Write("Enter the 4 elements of first matrix: ");
+		
+		for(int i=0;i<2;i++){
+		    var numList= new string[2];
+		    numList=Console.ReadLine().Split();
+		    for(int j=0;j<2;j++){
+	        	firstMatrix[i, j] = Convert.ToInt32(numList[j]); 
+	        	}
+	    	 }
+	    
+	    	Console.Write("Enter the 4 elements of second matrix: ");
```

Comment:
I see this implementation is for 2X2 matrix. Can we not have for N X N matrix because other codes in the folder are written for n X n ?